### PR TITLE
adding jctc pub

### PR DESCRIPTION
**Pull request by vivek-bala**

### CODE REVIEW

**Review by iparask — CHANGES_REQUESTED** [review]
Vivek, this is based on an old style. We have changed from then.

Can you check [here](https://raw.githubusercontent.com/radical-group/radical-group.github.io/master/_publications/paraskevakos2018task.md) and make the necessary changes? 

Thank you!

**Comment by vivek-bala** [comment]
Sure, will do. I merged my local branch with master and didn't see the change. Will convert it now.

**Comment by iparask** [comment]
I suggest you delete your local branch when not used. Also it does not adhere with the branching model for publications:

> When adding a publication to the webpage, create a branch named `publication/<pub-filename>` and create a pull request towards master of this repo. As soon as the request is merged, GitHub will render the page on the official website.

I do not really mind, but the model we have avoids merging issues definitely.

**Comment by vivek-bala** [comment]
> I suggest you delete your local branch when not used. Also it does not adhere with the branching model for publications

I see. I might be following some older guide. I have pushed with the same publications/vivek branch for this PR. But will follow it for any future changes. Hope that's okay for now.

File: _publications/shkurti2019cocomd.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
+title: "CoCo-MD: A Simple and Effective Method for the Enhanced Sampling of Conformational Space"
+collection: publications
+permalink: /publications/shkurti2019cocomd
+type: pub
+date: 2019-01-08
+author: "Ardita Shkurti, Ioanna Danai Styliari, Vivek Balasubramanian, Iain Bethune, Conrado Pedebos, Shantenu Jha, and Charles A. Laughton"
+venue: "Journal of Chemical Theory and Computation, 2019"
+paperurl: https://pubs.acs.org/doi/pdf/10.1021/acs.jctc.8b00657
```
Comment (iparask) [inline _publications/shkurti2019cocomd.md]: The link does not point to the publication site, but the pdf. It should point to where the paper is published.

**Review by iparask — CHANGES_REQUESTED** [review]

File: _publications/shkurti2019cocomd.md (hunk 1 above)
Comment (vivek-bala) [reply _publications/shkurti2019cocomd.md]: Done. Thanks for checking.

**Review by iparask — APPROVED** [review]

**Pull request merged**